extensions/wikia/ArticleVideo/scripts/recommended-video.jwplayer.js

```diff
@@ -118,6 +121,12 @@ require([
 	}
 
 	function init() {
+		var isInAutoplayGroup = window.Wikia.AbTest.inGroup('RECOMMENDED_VIDEO_AB', 'AUTOPLAY'),
```

Comment:
use https://github.com/Wikia/mobile-wiki/blob/d763536583337a228cfbb6d0eed4b85a2123e39e/app/modules/abtest.js#L22 instead of directly global var.